sampleapp/src/main/scala/de/up/hpi/informationsystems/sampleapp/TestApplication.scala

```diff
@@ -81,7 +81,7 @@ object TestApplication extends App {
 
   println()
   println("Projection of user relation:")
-  println(User.project(Seq(User.colFirstname, User.colLastname)).getOrElse(Seq.empty).pretty)
+  println(User.project(Set(User.colFirstname, User.colLastname): Set[UntypedColumnDef]).getOrElse(Set.empty))
```

Comment:
_optional_:
you could also use `Set[UntypedColumnDef](User.colFirstname, User.colLastname)`

Reply:
Oh right. That's neater.